### DISCUSSION TITLE
part-ii/2016-45/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#276] [Question comments]
Part II 2016 Differential Geometry: Paper 2, Section II, G 

https://questions.tripos.org/part-ii/2016-45/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a) the normal curvature along the line is zero, so $k_1\geq 0$, $k_2\leq 0$. Hence the Gaussian curvature is non positive at each point of $L$.

(b) $N = \frac{1}{\cosh(v)}\left(-\sin u,\cos u,-\sinh v\right)$. So the image of the Gauss map is $S^2$.

$\phi_u = \left(-\sinh(v)\sin(u),\sinh(v)\cos(u),1\right)$

$\phi_v = \left(\cosh(v)\cos(u),\cosh(v)\sin(u),0\right)$

$\phi_{uu} = \left(-\sinh(v)\cos(u),-\sinh(v)\sin(u),0\right)$

$\phi_{uv} = \left(-\cosh(v)\sin(u),\cosh(v)\cos(u),0\right)$

$\phi_{vv} = \left(\sinh(v)\cos(u),\sinh(v)\sin(u),0\right)$

$N = \frac{1}{\cosh(v)}\left(-\sin(u),\cos(u),-\sinh(v)\right)$

$e = 0 = F$.

$f = 1$.

$E = G = \cosh^2(v)$

$K = \frac{ef-g^2}{EF-G^2} = -\frac{1}{\cosh^4(v)}$

(c)

Line: fix $u$, let $v$ vary.




</details>
